src/BenchmarksApps/Websocket/Program.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Crank.EventSources;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace BenchmarkServer
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            Console.WriteLine($"Process ID: {Process.GetCurrentProcess().Id}");
+
+            BenchmarksEventSource.MeasureAspNetVersion();
```

Comment:
That's fine, but FYI crank now stores it automatically. I have seen it get skipped because of event pipes timing issues that I decide to let the crank server record it automatically.